vocabularies-gsq/gsq-alias.ttl

```diff
@@ -44,6 +44,7 @@ als:government
         als:mmol ,
         als:ogia ,
         als:treasury ;
+        als:wims-sample-id ;
```

Comment:
Why should wims-sample-id be included in this collection?  I may be wrong, but it doesn't appear consistent with the other concepts. A 'WIMS' concept would fit here, in my opinion, but might be premature before the system is named officially.

Reply:
WIMS is akin to MMOL, which is a system from another business group, hence it's inclusion here. 
We must deliver the SQL view by the 15th, and this is the final blocker for me to finalise the SQL. I don't think there is any chance WIMS will be renamed on completion

Reply:
Any inconsistency here is likely due to a lack of specificity in the other concepts. 
Take MMOL for example, there are many identifiers in that database, so MMOL as an alias source is useless. MMOL permit id, or mmol subblock id might be more relevant. 
WIMS sample id is the alias in the wims system, and calling this alias WIMS would make it unusable in geoprops since we can only specify and alias value.